tests/test_phirgen.py

```diff
@@ -463,3 +463,26 @@ def test_nullary_ops() -> None:
         "cop": "==",
         "args": [["tk_SCRATCH_BIT", 1], 1],  # evals to False
     }
+
+
+def test_condition_multiple_bits() -> None:
+    """From https://github.com/CQCL/pytket-phir/issues/215 ."""
+    n_bits = 3
+    c = Circuit(1, n_bits)
+    c.Rz(0.5, 0, condition_bits=list(range(n_bits)), condition_value=6)
+    phir = json.loads(pytket_to_phir(c))
+
+    assert phir["ops"][2] == {"//": "IF ([c[0], c[1], c[2]] == 6) THEN Rz(0.5) q[0];"}
+    assert phir["ops"][3]["condition"] == {
+        "cop": "&",
+        "args": [
+            {"cop": "==", "args": [["c", 0], 0]},
+            {
+                "cop": "&",
+                "args": [
+                    {"cop": "==", "args": [["c", 1], 1]},
+                    {"cop": "==", "args": [["c", 2], 1]},
```

Comment:
Note endianness of pytket: https://github.com/CQCL/pytket-phir/pull/162#discussion_r1559038837

> Need to be careful of endianness here. The conditional `value` in pytket is little-endian, e.g. a value of 2 on the bits `[a[0], a[1]]` means `a[0] == 0 && a[1] == 1`.